05-15 17:36:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "VALID" not found; SQL statement:
INSERT INTO customers(first_name,last_name, valid) values(?,?) [42122-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.table.Table.getColumn(Table.java:626)
	at org.h2.command.Parser.parseColumn(Parser.java:826)
	at org.h2.command.Parser.parseColumnList(Parser.java:811)
	at org.h2.command.Parser.parseInsert(Parser.java:1004)
	at org.h2.command.Parser.parsePrepared(Parser.java:394)
	at org.h2.command.Parser.parse(Parser.java:298)
	at org.h2.command.Parser.parse(Parser.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:235)
	at org.h2.engine.Session.prepareLocal(Session.java:436)
	at org.h2.engine.Session.prepareCommand(Session.java:379)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at org.springframework.jdbc.core.JdbcTemplate$SimplePreparedStatementCreator.createPreparedStatement(JdbcTemplate.java:1557)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:638)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:909)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:970)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:980)
	at h2app.Application.main(Application.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
05-15 17:36:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Column "VALID" not found; SQL statement:
INSERT INTO customers(first_name,last_name, valid) values(?,?,?) [42122-175]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:332)
	at org.h2.message.DbException.get(DbException.java:172)
	at org.h2.message.DbException.get(DbException.java:149)
	at org.h2.table.Table.getColumn(Table.java:626)
	at org.h2.command.Parser.parseColumn(Parser.java:826)
	at org.h2.command.Parser.parseColumnList(Parser.java:811)
	at org.h2.command.Parser.parseInsert(Parser.java:1004)
	at org.h2.command.Parser.parsePrepared(Parser.java:394)
	at org.h2.command.Parser.parse(Parser.java:298)
	at org.h2.command.Parser.parse(Parser.java:270)
	at org.h2.command.Parser.prepareCommand(Parser.java:235)
	at org.h2.engine.Session.prepareLocal(Session.java:436)
	at org.h2.engine.Session.prepareCommand(Session.java:379)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1138)
	at org.h2.jdbc.JdbcPreparedStatement.<init>(JdbcPreparedStatement.java:70)
	at org.h2.jdbc.JdbcConnection.prepareStatement(JdbcConnection.java:267)
	at org.springframework.jdbc.core.JdbcTemplate$SimplePreparedStatementCreator.createPreparedStatement(JdbcTemplate.java:1557)
	at org.springframework.jdbc.core.JdbcTemplate.execute(JdbcTemplate.java:638)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:909)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:970)
	at org.springframework.jdbc.core.JdbcTemplate.update(JdbcTemplate.java:980)
	at h2app.Application.main(Application.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
